index.html

```diff
@@ -111,6 +111,29 @@ <h2 class="ui divider horizontal">Services utilisant les API</h2>
   </p>
 </div>
 
+<section id="diff-data-api" class="ui container grid">
+  <div class="item eight wide column">
```

Comment:
What is this `item` class used for? If it's just for your CSS adjustments, target `.column` and get rid of this specific `item` thing.